src/main/java/mate/academy/spring/model/dto/response/AbstractResponseDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public abstract class AbstractResponseDto {
```

Comment:
```suggestion
public abstract class UserTicketsResponseDto {
```
it's let's make this name more specific